tests/pipelines/pixart_alpha/test_pixart.py

```diff
@@ -332,7 +338,7 @@ def tearDown(self):
         torch.cuda.empty_cache()
 
     def test_pixart_1024(self):
-        generator = torch.manual_seed(0)
```

Comment:
Shouldn't it by default create it on CPU?

Reply:
Yes. I just changed it to match how the generator is created in the other tests.